packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [35.1.0]
+
+### Added
+
+- Add optional `topBid` property to the `NftMetadata` type. This property must be of type `TopBid`. ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Add optional `floorAsk` property to the `TokenCollection` type. This property must be of type `FloorAskCollection`. ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Add linea mainnet support to nft detection supported networks ([#4515](https://github.com/MetaMask/core/pull/4515))
+- Fetch NFT collections data from the NFT-API `getCollections` API when calling the `detectNfts` method of `NftDetectionController`, and the `updateNftMetadata` and `watchNft` methods of `NftController`. ([#4443](https://github.com/MetaMask/core/pull/4443))
+
+### Changed
+
```

Comment:
This looks like it might be a change in behavior for the affected class methods, not an example of the package API's surface area being extended (see https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md#additions).

Would you agree that we are returning more data to the consumer (or to other intermediate call sites), but we're not expanding the interface with which the consumer is expected to interact when calling these methods?

```suggestion

### Changed

- Fetch NFT collections data from the NFT-API `getCollections` API when calling the `detectNfts` method of `NftDetectionController`, and the `updateNftMetadata` and `watchNft` methods of `NftController`. ([#4443](https://github.com/MetaMask/core/pull/4443))
```

Reply:
Agree, when clients want to interact with the new responses they will interact with 
`import { Nft } from '@metamask/assets-controllers';` where all the newly fetched data is optional and new access to the new data will treat new values as optional;

Reply:
If so, shouldn't this entry be recategorized as a `Changed` entry?

Reply:
Hmmm, I dnt have a strong opinion on this, I thought we are in the case of adding optional properties to an existing type in the Additions right? [section](https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md#additions)

Where clients in future would also treat this new data as optional; so we are not creating any new dependencies.
Just adding few more optional values that if exists should be displayed, if not, they will be ignored.

I can update it to `Changed` if you think that makes more sense

Reply:
The type changes and the method behavior changes seem like separate updates. How would you feel about splitting this up into an Added and a Changed entry?

```suggestion
- The `Collection` type is expanded to include the following 'string'-type optional properties: `contractDeployedAt`, `creator`, `ownerCount`, and an optional property `topBid` of the type `TopBid & { sourceDomain?: string; }`. ([#4443](https://github.com/MetaMask/core/pull/4443))

### Changed

- Fetch NFT collections data from the OpenSea API `Get Collections` endpoint when calling the `detectNfts` method of `NftDetectionController`, and the `updateNftMetadata` and `watchNft` methods of `NftController`. ([#4443](https://github.com/MetaMask/core/pull/4443))
```

Reply:
Updated 🙏

Reply:
I only put NFT-API instead of OpenSea API because it is more accurate;